src/Tvheadend.cpp

```diff
@@ -2459,6 +2459,21 @@ void CTvheadend::ParseRecordingAddOrUpdate ( htsmsg_t *msg, bool bAdd )
     rec.SetDescription(str);
   else if ((str = htsmsg_get_str(msg, "summary")) != NULL)
     rec.SetDescription(str);
+  /* 
+    Due to changes in HTSP v32, try to use subtitle 
+    as the description if the description and summary
+    failed. Clear the subtitle afterwards  to avoid
+    duplicate information being displayed.
+
+    This was done by TVHeadend prior to HTSP v32.
+  */
+  else if (rec.GetDescription().empty() && m_conn->GetProtocol() >= 32)
```

Comment:
Why `else if`and not just `if`? you might have gotten empty desc or summary above...